ui/components/component-library/icon/icon.types.ts

```diff
@@ -61,6 +61,7 @@ export enum IconName {
   Connect = 'connect',
   CopySuccess = 'copy-success',
   Copy = 'copy',
+  Custody = 'custody',
```

Comment:
The snaps SDK exports a type that contains this, so we can't remove this without causing typescript errors

Reply:
There is also an instance of this being used in a test that has not been removed

![Screenshot 2025-01-28 at 11 26 36 AM](https://github.com/user-attachments/assets/e94e21c2-9f98-475e-83ee-870e3c62ea0f)